Sensor Fusion/broadcastreceiver/filterloophandler.cpp

```diff
@@ -2,7 +2,7 @@
 #include <QTimer>
 #include "filterloophandler.h"
 #include "sensorvalues.h"
-
+#include <QFile>
```

Comment:
Maybe we should remove the writing to files?
As it is more of an debug/presentation part and bit misplaced where there are? We don't need them anymore?

@zpettzon @alirezatashvir